### DISCUSSION TITLE
Asking for the meaning of "rotary_percentage" in Rotary Position Embedding.py

### DISCUSSION THREAD

**Discussion by YOONSEOKHEO** [NVIDIA-NeMo/NeMo#8588] [Q&A]
Hi. 
I am digging into somewhere in Nemo nowadays. 
I cannot follow up any cases for downscaling the "rotary_dim" by using "config.rotary_percentage".
Here is the URL: https://github.com/NVIDIA/NeMo/blob/d2283e3620cd7f99dbe29fdf079757ab9f6cdf01/nemo/collections/nlp/modules/common/megatron/language_model.py#L565

Does anyone share the concept of this variable and its process? 
I am very appreciating your comments in advance.

**Comment by ashvinnihalani** [comment]
+1 Can we get an idea of what this does? I know it modifies the head dimension but a paper or something of how this applies would be greatly appreciated
